OpenSim/Simulation/InverseKinematicsSolver.cpp

```diff
@@ -233,9 +233,10 @@ void InverseKinematicsSolver::setupGoals(SimTK::State &s)
             const SimTK::MobilizedBody& mobod =
                 marker.getReferenceFrame().getMobilizedBody();
 
-            const SimTK::Transform& X_BF = marker.getReferenceFrame().getTransformInMobilizedBody();
+            const SimTK::Transform& X_BF = marker.getReferenceFrame().findTransformInBaseFrame();
```

Comment:
If the method is `findSomething()` it presumably is calculating something rather than returning a reference. So the return variable shouldn't be a reference, just a local variable.